bokehjs/test/integration/axes.ts

```diff
@@ -146,6 +150,10 @@ import {radians} from "@bokehjs/core/util/math"
       await plot({axis_label: multiline_axis_label, axis_label_text_align: "right"}, {minor_size: 100})
     })
 
+    it("should support math text in axis_label", async () => {
+      await plot({axis_label: new MathText({text: "\\sin(x)"})}, {minor_size: 100})
```

Comment:
Tests must be self contained (among other things), so in this case they must not require access to the internet (e.g. I would like to test things on a plane), so you will need to figure out how to test this accordingly, e.g. serve appropriate network resources locally.

Reply:
Users will be loading mathjax from the internet so a test must test this. A conditional test you can easily omit locally (analogue of pytest marker) would be appropriate. Self contained is good general guidance but absolute not requireable in every situation.

Reply:
We need to test the internals of bokehjs robustly, regardless of the internet connection, CDN availability, etc. We don't want the tests to fail, because the library changed or got temporarily corrupted in some way. Moreover, we need to test the remote code loader's behavior, but we need to do this in a controlled environment, not over the internet (devtools provide tooling for this). However, we also need to have end-to-end tests, where we test with the actual CDN, but not here, not when testing the rendering engine. Those need to be separate tests, perhaps even on a completely new test track.

Same applies e.g. to tile rendering. Currently we don't have any bokehjs integration tests there (well, one skipped), because those tiles change constantly and tests would fail constantly as well. So once again, we need to have a controlled testing environment for testing, in particular rendering of, tiles, and then separately test whether bokeh works with actual tile providers.